apps/meteor/app/otr/client/rocketchat.otr.room.js

```diff
@@ -275,11 +285,11 @@ OTR.Room = class {
 
 				(async () => {
 					const { username } = await Presence.get(data.userId);
-					if (data.refresh && this.established.get()) {
+					if (data.refresh) {
```

Comment:
shouldn't we check the established state?

Reply:
Done!